### DISCUSSION TITLE
Undefined function 'str_replace'

### DISCUSSION THREAD

**Discussion by jonycp** [symfony/symfony#50674] [Q&A]
I am new to symfony. I have the symfony-cli on my mac and ran symfony new myproject. When I try to add to the skeleton, using  PHPStorm,  it flags str_replace() as an Undefined function. Auto-complete brings up only functions such as mb_stripos(), mb_str_split(), etc.  

When I make an empty project in PHP Storm, the functions, str_replace() for example, are available.  

Is there something in the skeleton build that restricts or repackages the available php functions?

Thank you.

**Comment by ghost** [comment]
No, its not Symfony related. It's something wrong with your IDE.

**Comment by jonycp** [comment]
Ah, ok. Thank you!  I will take this up with the IDE vendor.

Much appreciated.

Best wishes,

Jon

**Comment by jonycp** [comment]
To finish this thread, the issue was Java installation. Updating this fixed the problem. 

Thanks again for your quick reply.

**Comment by jonycp** [comment]
Closed
